cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlUtils.java

```diff
@@ -41,7 +41,7 @@
   private static final Set<String> KAFKA_ASSIGNER_GOALS =
       Collections.unmodifiableSet(new HashSet<>(Arrays.asList(KafkaAssignerEvenRackAwareGoal.class.getSimpleName(),
                                                               KafkaAssignerDiskUsageDistributionGoal.class.getSimpleName())));
-  public static final String OPERATION_LOGGER = "log4j.logger.com.linkedin.kafka.cruisecontrol.CruiseControlOperationLog";
+  public static final String OPERATION_LOGGER = "operationLogger";
```

Comment:
Shouldn't the name of the logger match the name we use in `config/log4j.properties`? In this case `operationLogger` -> `operationLog`?